pkg/command/output.go

```diff
@@ -96,17 +97,21 @@ func recursiveMerge(i1, i2 interface{}) interface{} {
 }
 
 //PrintOutputWithType receives an interface and dump the data using the --output flag.
-//ATM only json or jsonpath. In the future yaml
+//ATM only json, yaml, or jsonpath.
```

Comment:
Nit (proper godoc comment): add a single space between the `//` and the following text.

Reply:
Agreed. I will add that. In general, I'm tempted to review a few more comments in that file while I'm at it.